File: Section Reports/Layout/SubReport/C#/SubReport/rptNestedChildSub.cs
namespace GrapeCity.ActiveReports.Samples.SubReport
{
	/// <summary>
	/// Description of the outline of the rptNestedChildSub
	/// 
	/// </summary>
	public partial class rptNestedChildSub : GrapeCity.ActiveReports.SectionReport
	{
		public rptNestedChildSub()
		{
			//
			// ActiveReport designer support is required.
			//
			//
			InitializeComponent();
			Data.OleDBDataSource dS = new Data.OleDBDataSource();			
			dS.ConnectionString = Properties.Resources.ConnectionString;			
			dS.SQL = "Select * from customers where customerID = '<%CustomerID%>' ";
			DataSource = dS;
		}
	}
}

File: Section Reports/Layout/SubReport/VB.NET/SubReport/rptNestedChildSub.vb
Public Class rptNestedChildSub
	Inherits GrapeCity.ActiveReports.SectionReport
	Public Sub New()
		MyBase.New()
		InitializeComponent()
		Dim dS As New Data.OleDBDataSource()
		dS.ConnectionString = My.Resources.ConnectionString
		dS.SQL = "Select * from customers where customerID = '<%CustomerID%>'"
		DataSource = dS
	End Sub
End Class
